### PR TITLE
Add update delete

### PR DESCRIPTION
**Pull request by syncrou**

### CODE REVIEW

**Comment by syncrou** [comment]
@miq-bot assign @bdunne

**Comment by syncrou** [comment]
@miq-bot add_label enhancement

File: spec/inventory_source_spec.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@
   let(:raw_instance)        { build(:response_instance, :klass => described_class) }
 
   include_examples "Collection Methods"
-  include_examples "Crud Methods"
```
Comment (syncrou) [inline spec/inventory_source_spec.rb]: Consistency with `activity_stream` - `inventory_sources` does not support `post`, `put` or `patch` so I think we should pull out the shared examples and implement CRUD on those 'special' endpoints specific to our needs.

Comment (bdunne) [inline spec/inventory_source_spec.rb]: ok

**Comment by syncrou** [comment]
@bdunne - All changes requested are finished

**Pull request merged**